test/tests/functional/pbs_sister_mom_crashes.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestMomCrash(TestFunctional):
+    """
+    This test suite tests the sister mom
+    """
+    @timeout(360)
+    def test_interactive_job(self):
+        """
+        This test suite verifies that sister mom
+        doesn't crash in an interactive session
+        """
+        # Skip test if number of mom provided is not equal to two
+        if len(self.moms) != 2:
+            self.skipTest("test requires atleast two MoMs as input, " +
+                          "use -p moms=<mom1:mom2>")
+        for key in self.moms:
+            sister_mom = key
+        j = Job(TEST_USER, attrs={ATTR_inter: '',
+                                  'Resource_List.select': '2',
+                                  'Resource_List.place': 'scatter'})
+        j.interactive_script = [('pbsdsh dd if=/dev/zero of=/dev/null', '.*')]
```

Comment:
Can just have a batch job that calls this pbsdsh, so in that way, you won't need to look for a "sleep" in pexpect...

Reply:
@Bhagat-Rajput 
I agree with Al. Can you please change your PTL test case so as just to have batch job instead of interactive job.